frontend/public/components/utils/_popover.scss

```diff
@@ -0,0 +1,4 @@
+.co-divider-padding {
```

Comment:
Why do we need to override the PF styles here? What does it look like without this change?